app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Quote Results</h1>
+
+<h3>Total Count: <%= @quotes[:total_count] %></h3>
+
```

Comment:
So as you mentioned in your self reflection, we definitely don't want to use hash notation in our view and want objects instead. So see my comment in your facade.